ui/_webSrc/vite.config.ts

```diff
@@ -0,0 +1,21 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+
+// https://vitejs.dev/config/
+export default defineConfig({
```

Comment:
a few options to make files easier to manage without knowing how to adjust python